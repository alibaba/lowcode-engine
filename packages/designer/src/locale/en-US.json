{
  "copy": "Copy",
  "remove": "Remove",
  "hide": "Hide",
  "lock": "Lock",
  "unlock": "Unlock",
  "Condition Group": "Condition Group",
  "No opened document": "No opened document, open some document to editing"
}
