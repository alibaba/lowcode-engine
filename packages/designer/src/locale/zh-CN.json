{
  "copy": "复制",
  "remove": "删除",
  "hide": "隐藏",
  "lock": "锁定",
  "unlock": "解锁",
  "Condition Group": "条件组",
  "No opened document": "没有打开的页面，请选择页面打开编辑"
}
