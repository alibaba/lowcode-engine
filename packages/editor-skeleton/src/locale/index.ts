import { createIntl } from '@alilc/lowcode-editor-core';
import enUS from './en-US.json';
import zhCN from './zh-CN.json';

const { intl, intlNode, getLocale, setLocale } = createIntl({
  'en-US': enUS,
  'zh-CN': zhCN,
});

export { intl, intlNode, getLocale, setLocale };
