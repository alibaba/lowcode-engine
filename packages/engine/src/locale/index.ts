import { createIntl } from '@alilc/lowcode-editor-core';
import enUS from './en-US.json';
import zhCN from './zh-CN.json';

const { intl, getLocale } = createIntl?.({
  'en-US': enUS,
  'zh-CN': zhCN,
}) || {
  intl: (id) => {
    return zhCN[id];
  },
};

export { intl, enUS, zhCN, getLocale };
