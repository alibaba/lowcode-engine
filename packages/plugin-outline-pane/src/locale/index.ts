import { createIntl } from '@ali/lowcode-editor-core';
import en_US from './en-US.json';
import zh_CN from './zh-CN.json';

const { intl, intlNode, getLocale, setLocale } = createIntl({
  'en-US': en_US,
  'zh-CN': zh_CN,
});

export { intl, intlNode, getLocale, setLocale };
