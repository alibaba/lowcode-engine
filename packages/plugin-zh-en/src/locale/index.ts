import { createIntl } from '@ali/lowcode-editor-core';
import en_US from './en-US.json';
import zh_CN from './zh-CN.json';

const { intl, getLocale, setLocale } = createIntl({
  'en-US': en_US,
  'zh-CN': zh_CN,
});

export { intl, getLocale, setLocale };
