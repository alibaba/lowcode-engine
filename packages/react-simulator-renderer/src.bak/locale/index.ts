import { createElement } from 'react';
import enUS from './en-US.json';
import zhCN from './zh-CN.json';

const instance: Record<string, Record<string, string>> = {
  'zh-CN': zhCN as Record<string, string>,
  'en-US': enUS as Record<string, string>,
};

export function createIntl(locale: string = 'zh-CN') {
  const intl = (id: string) => {
    return instance[locale]?.[id] || id;
  };

  const intlNode = (id: string) => createElement('span', instance[locale]?.[id] || id);

  return {
    intl,
    intlNode,
  };
}
