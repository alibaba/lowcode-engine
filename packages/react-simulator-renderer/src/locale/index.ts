import { createElement } from 'react';
import enUS from './en-US.json';
import zhCN from './zh-CN.json';

const instance: Record<string, Record<string, string>> = {
  'zh-CN': zhCN as Record<string, string>,
  'en-US': enUS as Record<string, string>,
};

export function createIntl(locale: string = 'zh-CN') {
  const intl = (id: string) => {
    return instance[locale][id];
  };

  const intlNode = (id: string) => createElement('span', instance[locale][id]);

  return {
    intl,
    intlNode,
  };
}
