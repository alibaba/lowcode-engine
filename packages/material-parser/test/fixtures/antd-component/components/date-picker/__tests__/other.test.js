import React from 'react';
import { mount, render } from 'enzyme';
import moment from 'moment';
import DatePicker from '..';
import LocaleProvider from '../../locale-provider';
import locale from '../../locale-provider/zh_CN';

const { MonthPicker, WeekPicker } = DatePicker;

describe('Picker format by locale', () => {
  const myLocale = {
    ...locale,
    DatePicker: {
      ...locale.DatePicker,
      dateFormat: 'YYYY 年 M 月 D 日',
      dateTimeFormat: 'YYYY 年 M 月 D 日 H 时 m 分 s 秒',
      weekFormat: 'YYYY 年 W 周',
      monthFormat: 'YYYY 年 M 月',
    },
  };

  const date = moment('2000-01-01', 'YYYY-MM-DD');
  function matchPicker(name, Picker, props) {
    it(name, () => {
      const wrapper = mount(
        <LocaleProvider locale={myLocale}>
          <Picker value={date} {...props} />
        </LocaleProvider>,
      );

      expect(wrapper.render()).toMatchSnapshot();
    });
  }

  matchPicker('date', DatePicker);
  matchPicker('dateTime', DatePicker, { showTime: true });
  matchPicker('week', WeekPicker);
  matchPicker('month', MonthPicker);
});

describe('MonthPicker and WeekPicker', () => {
  it('render MonthPicker', () => {
    const birthday = moment('2000-01-01', 'YYYY-MM-DD').locale('zh-cn');
    const wrapper = mount(<MonthPicker open />);
    wrapper.setProps({ value: birthday });
    expect(
      render(
        wrapper
          .find('Trigger')
          .instance()
          .getComponent(),
      ),
    ).toMatchSnapshot();
  });

  it('render WeekPicker', () => {
    const birthday = moment('2000-01-01', 'YYYY-MM-DD').locale('zh-cn');
    const wrapper = mount(<WeekPicker open />);
    wrapper.setProps({ value: birthday });
    expect(
      render(
        wrapper
          .find('Trigger')
          .instance()
          .getComponent(),
      ),
    ).toMatchSnapshot();
  });
});
