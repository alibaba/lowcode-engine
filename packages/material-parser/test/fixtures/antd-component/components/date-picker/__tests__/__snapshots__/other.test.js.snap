// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MonthPicker and WeekPicker render MonthPicker 1`] = `
<div>
  <div
    class="ant-picker-dropdown"
    style="opacity:0"
  >
    <div
      class="ant-picker-panel-container"
    >
      <div
        class="ant-picker-panel ant-picker-panel-focused"
        tabindex="-1"
      >
        <div
          class="ant-picker-month-panel"
        >
          <div
            class="ant-picker-header"
          >
            <button
              class="ant-picker-header-super-prev-btn"
              tabindex="-1"
              type="button"
            >
              «
            </button>
            <div
              class="ant-picker-header-view"
            >
              <button
                class="ant-picker-year-btn"
                type="button"
              >
                2000
              </button>
            </div>
            <button
              class="ant-picker-header-super-next-btn"
              tabindex="-1"
              type="button"
            >
              »
            </button>
          </div>
          <div
            class="ant-picker-body"
          >
            <table
              class="ant-picker-content"
            >
              <tbody>
                <tr>
                  <td
                    class="ant-picker-cell ant-picker-cell-in-view ant-picker-cell-selected"
                    title="2000-01"
                  >
                    <div
                      class="ant-picker-cell-inner"
                    >
                      Jan
                    </div>
                  </td>
                  <td
                    class="ant-picker-cell ant-picker-cell-in-view"
                    title="2000-02"
                  >
                    <div
                      class="ant-picker-cell-inner"
                    >
                      Feb
                    </div>
                  </td>
                  <td
                    class="ant-picker-cell ant-picker-cell-in-view"
                    title="2000-03"
                  >
                    <div
                      class="ant-picker-cell-inner"
                    >
                      Mar
                    </div>
                  </td>
                </tr>
                <tr>
                  <td
                    class="ant-picker-cell ant-picker-cell-in-view"
                    title="2000-04"
                  >
                    <div
                      class="ant-picker-cell-inner"
                    >
                      Apr
                    </div>
                  </td>
                  <td
                    class="ant-picker-cell ant-picker-cell-in-view"
                    title="2000-05"
                  >
                    <div
                      class="ant-picker-cell-inner"
                    >
                      May
                    </div>
                  </td>
                  <td
                    class="ant-picker-cell ant-picker-cell-in-view"
                    title="2000-06"
                  >
                    <div
                      class="ant-picker-cell-inner"
                    >
                      Jun
                    </div>
                  </td>
                </tr>
                <tr>
                  <td
                    class="ant-picker-cell ant-picker-cell-in-view"
                    title="2000-07"
                  >
                    <div
                      class="ant-picker-cell-inner"
                    >
                      Jul
                    </div>
                  </td>
                  <td
                    class="ant-picker-cell ant-picker-cell-in-view"
                    title="2000-08"
                  >
                    <div
                      class="ant-picker-cell-inner"
                    >
                      Aug
                    </div>
                  </td>
                  <td
                    class="ant-picker-cell ant-picker-cell-in-view"
                    title="2000-09"
                  >
                    <div
                      class="ant-picker-cell-inner"
                    >
                      Sep
                    </div>
                  </td>
                </tr>
                <tr>
                  <td
                    class="ant-picker-cell ant-picker-cell-in-view"
                    title="2000-10"
                  >
                    <div
                      class="ant-picker-cell-inner"
                    >
                      Oct
                    </div>
                  </td>
                  <td
                    class="ant-picker-cell ant-picker-cell-in-view"
                    title="2000-11"
                  >
                    <div
                      class="ant-picker-cell-inner"
                    >
                      Nov
                    </div>
                  </td>
                  <td
                    class="ant-picker-cell ant-picker-cell-in-view"
                    title="2000-12"
                  >
                    <div
                      class="ant-picker-cell-inner"
                    >
                      Dec
                    </div>
                  </td>
                </tr>
              </tbody>
            </table>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`MonthPicker and WeekPicker render WeekPicker 1`] = `
<div>
  <div
    class="ant-picker-dropdown"
    style="opacity:0"
  >
    <div
      class="ant-picker-panel-container"
    >
      <div
        class="ant-picker-panel ant-picker-panel-focused"
        tabindex="-1"
      >
        <div
          class="ant-picker-week-panel"
        >
          <div
            class="ant-picker-header"
          >
            <button
              class="ant-picker-header-super-prev-btn"
              tabindex="-1"
              type="button"
            >
              <span
                class="ant-picker-super-prev-icon"
              />
            </button>
            <button
              class="ant-picker-header-prev-btn"
              tabindex="-1"
              type="button"
            >
              <span
                class="ant-picker-prev-icon"
              />
            </button>
            <div
              class="ant-picker-header-view"
            >
              <button
                class="ant-picker-month-btn"
                tabindex="-1"
                type="button"
              >
                Jan
              </button>
              <button
                class="ant-picker-year-btn"
                tabindex="-1"
                type="button"
              >
                2000
              </button>
            </div>
            <button
              class="ant-picker-header-next-btn"
              tabindex="-1"
              type="button"
            >
              <span
                class="ant-picker-next-icon"
              />
            </button>
            <button
              class="ant-picker-header-super-next-btn"
              tabindex="-1"
              type="button"
            >
              <span
                class="ant-picker-super-next-icon"
              />
            </button>
          </div>
          <div
            class="ant-picker-body"
          >
            <table
              class="ant-picker-content"
            >
              <thead>
                <tr>
                  <th />
                  <th>
                    Su
                  </th>
                  <th>
                    Mo
                  </th>
                  <th>
                    Tu
                  </th>
                  <th>
                    We
                  </th>
                  <th>
                    Th
                  </th>
                  <th>
                    Fr
                  </th>
                  <th>
                    Sa
                  </th>
                </tr>
              </thead>
              <tbody>
                <tr
                  class="ant-picker-week-panel-row ant-picker-week-panel-row-selected"
                >
                  <td
                    class="ant-picker-cell ant-picker-cell-week"
                  >
                    1
                  </td>
                  <td
                    class="ant-picker-cell"
                    title="1999-12-26"
                  >
                    <div
                      class="ant-picker-cell-inner"
                    >
                      26
                    </div>
                  </td>
                  <td
                    class="ant-picker-cell"
                    title="1999-12-27"
                  >
                    <div
                      class="ant-picker-cell-inner"
                    >
                      27
                    </div>
                  </td>
                  <td
                    class="ant-picker-cell"
                    title="1999-12-28"
                  >
                    <div
                      class="ant-picker-cell-inner"
                    >
                      28
                    </div>
                  </td>
                  <td
                    class="ant-picker-cell"
                    title="1999-12-29"
                  >
                    <div
                      class="ant-picker-cell-inner"
                    >
                      29
                    </div>
                  </td>
                  <td
                    class="ant-picker-cell"
                    title="1999-12-30"
                  >
                    <div
                      class="ant-picker-cell-inner"
                    >
                      30
                    </div>
                  </td>
                  <td
                    class="ant-picker-cell"
                    title="1999-12-31"
                  >
                    <div
                      class="ant-picker-cell-inner"
                    >
                      31
                    </div>
                  </td>
                  <td
                    class="ant-picker-cell ant-picker-cell-in-view ant-picker-cell-selected"
                    title="2000-01-01"
                  >
                    <div
                      class="ant-picker-cell-inner"
                    >
                      1
                    </div>
                  </td>
                </tr>
                <tr
                  class="ant-picker-week-panel-row"
                >
                  <td
                    class="ant-picker-cell ant-picker-cell-week"
                  >
                    2
                  </td>
                  <td
                    class="ant-picker-cell ant-picker-cell-in-view"
                    title="2000-01-02"
                  >
                    <div
                      class="ant-picker-cell-inner"
                    >
                      2
                    </div>
                  </td>
                  <td
                    class="ant-picker-cell ant-picker-cell-in-view"
                    title="2000-01-03"
                  >
                    <div
                      class="ant-picker-cell-inner"
                    >
                      3
                    </div>
                  </td>
                  <td
                    class="ant-picker-cell ant-picker-cell-in-view"
                    title="2000-01-04"
                  >
                    <div
                      class="ant-picker-cell-inner"
                    >
                      4
                    </div>
                  </td>
                  <td
                    class="ant-picker-cell ant-picker-cell-in-view"
                    title="2000-01-05"
                  >
                    <div
                      class="ant-picker-cell-inner"
                    >
                      5
                    </div>
                  </td>
                  <td
                    class="ant-picker-cell ant-picker-cell-in-view"
                    title="2000-01-06"
                  >
                    <div
                      class="ant-picker-cell-inner"
                    >
                      6
                    </div>
                  </td>
                  <td
                    class="ant-picker-cell ant-picker-cell-in-view"
                    title="2000-01-07"
                  >
                    <div
                      class="ant-picker-cell-inner"
                    >
                      7
                    </div>
                  </td>
                  <td
                    class="ant-picker-cell ant-picker-cell-in-view"
                    title="2000-01-08"
                  >
                    <div
                      class="ant-picker-cell-inner"
                    >
                      8
                    </div>
                  </td>
                </tr>
                <tr
                  class="ant-picker-week-panel-row"
                >
                  <td
                    class="ant-picker-cell ant-picker-cell-week"
                  >
                    3
                  </td>
                  <td
                    class="ant-picker-cell ant-picker-cell-in-view"
                    title="2000-01-09"
                  >
                    <div
                      class="ant-picker-cell-inner"
                    >
                      9
                    </div>
                  </td>
                  <td
                    class="ant-picker-cell ant-picker-cell-in-view"
                    title="2000-01-10"
                  >
                    <div
                      class="ant-picker-cell-inner"
                    >
                      10
                    </div>
                  </td>
                  <td
                    class="ant-picker-cell ant-picker-cell-in-view"
                    title="2000-01-11"
                  >
                    <div
                      class="ant-picker-cell-inner"
                    >
                      11
                    </div>
                  </td>
                  <td
                    class="ant-picker-cell ant-picker-cell-in-view"
                    title="2000-01-12"
                  >
                    <div
                      class="ant-picker-cell-inner"
                    >
                      12
                    </div>
                  </td>
                  <td
                    class="ant-picker-cell ant-picker-cell-in-view"
                    title="2000-01-13"
                  >
                    <div
                      class="ant-picker-cell-inner"
                    >
                      13
                    </div>
                  </td>
                  <td
                    class="ant-picker-cell ant-picker-cell-in-view"
                    title="2000-01-14"
                  >
                    <div
                      class="ant-picker-cell-inner"
                    >
                      14
                    </div>
                  </td>
                  <td
                    class="ant-picker-cell ant-picker-cell-in-view"
                    title="2000-01-15"
                  >
                    <div
                      class="ant-picker-cell-inner"
                    >
                      15
                    </div>
                  </td>
                </tr>
                <tr
                  class="ant-picker-week-panel-row"
                >
                  <td
                    class="ant-picker-cell ant-picker-cell-week"
                  >
                    4
                  </td>
                  <td
                    class="ant-picker-cell ant-picker-cell-in-view"
                    title="2000-01-16"
                  >
                    <div
                      class="ant-picker-cell-inner"
                    >
                      16
                    </div>
                  </td>
                  <td
                    class="ant-picker-cell ant-picker-cell-in-view"
                    title="2000-01-17"
                  >
                    <div
                      class="ant-picker-cell-inner"
                    >
                      17
                    </div>
                  </td>
                  <td
                    class="ant-picker-cell ant-picker-cell-in-view"
                    title="2000-01-18"
                  >
                    <div
                      class="ant-picker-cell-inner"
                    >
                      18
                    </div>
                  </td>
                  <td
                    class="ant-picker-cell ant-picker-cell-in-view"
                    title="2000-01-19"
                  >
                    <div
                      class="ant-picker-cell-inner"
                    >
                      19
                    </div>
                  </td>
                  <td
                    class="ant-picker-cell ant-picker-cell-in-view"
                    title="2000-01-20"
                  >
                    <div
                      class="ant-picker-cell-inner"
                    >
                      20
                    </div>
                  </td>
                  <td
                    class="ant-picker-cell ant-picker-cell-in-view"
                    title="2000-01-21"
                  >
                    <div
                      class="ant-picker-cell-inner"
                    >
                      21
                    </div>
                  </td>
                  <td
                    class="ant-picker-cell ant-picker-cell-in-view"
                    title="2000-01-22"
                  >
                    <div
                      class="ant-picker-cell-inner"
                    >
                      22
                    </div>
                  </td>
                </tr>
                <tr
                  class="ant-picker-week-panel-row"
                >
                  <td
                    class="ant-picker-cell ant-picker-cell-week"
                  >
                    5
                  </td>
                  <td
                    class="ant-picker-cell ant-picker-cell-in-view"
                    title="2000-01-23"
                  >
                    <div
                      class="ant-picker-cell-inner"
                    >
                      23
                    </div>
                  </td>
                  <td
                    class="ant-picker-cell ant-picker-cell-in-view"
                    title="2000-01-24"
                  >
                    <div
                      class="ant-picker-cell-inner"
                    >
                      24
                    </div>
                  </td>
                  <td
                    class="ant-picker-cell ant-picker-cell-in-view"
                    title="2000-01-25"
                  >
                    <div
                      class="ant-picker-cell-inner"
                    >
                      25
                    </div>
                  </td>
                  <td
                    class="ant-picker-cell ant-picker-cell-in-view"
                    title="2000-01-26"
                  >
                    <div
                      class="ant-picker-cell-inner"
                    >
                      26
                    </div>
                  </td>
                  <td
                    class="ant-picker-cell ant-picker-cell-in-view"
                    title="2000-01-27"
                  >
                    <div
                      class="ant-picker-cell-inner"
                    >
                      27
                    </div>
                  </td>
                  <td
                    class="ant-picker-cell ant-picker-cell-in-view"
                    title="2000-01-28"
                  >
                    <div
                      class="ant-picker-cell-inner"
                    >
                      28
                    </div>
                  </td>
                  <td
                    class="ant-picker-cell ant-picker-cell-in-view"
                    title="2000-01-29"
                  >
                    <div
                      class="ant-picker-cell-inner"
                    >
                      29
                    </div>
                  </td>
                </tr>
                <tr
                  class="ant-picker-week-panel-row"
                >
                  <td
                    class="ant-picker-cell ant-picker-cell-week"
                  >
                    6
                  </td>
                  <td
                    class="ant-picker-cell ant-picker-cell-in-view"
                    title="2000-01-30"
                  >
                    <div
                      class="ant-picker-cell-inner"
                    >
                      30
                    </div>
                  </td>
                  <td
                    class="ant-picker-cell ant-picker-cell-in-view"
                    title="2000-01-31"
                  >
                    <div
                      class="ant-picker-cell-inner"
                    >
                      31
                    </div>
                  </td>
                  <td
                    class="ant-picker-cell"
                    title="2000-02-01"
                  >
                    <div
                      class="ant-picker-cell-inner"
                    >
                      1
                    </div>
                  </td>
                  <td
                    class="ant-picker-cell"
                    title="2000-02-02"
                  >
                    <div
                      class="ant-picker-cell-inner"
                    >
                      2
                    </div>
                  </td>
                  <td
                    class="ant-picker-cell"
                    title="2000-02-03"
                  >
                    <div
                      class="ant-picker-cell-inner"
                    >
                      3
                    </div>
                  </td>
                  <td
                    class="ant-picker-cell"
                    title="2000-02-04"
                  >
                    <div
                      class="ant-picker-cell-inner"
                    >
                      4
                    </div>
                  </td>
                  <td
                    class="ant-picker-cell"
                    title="2000-02-05"
                  >
                    <div
                      class="ant-picker-cell-inner"
                    >
                      5
                    </div>
                  </td>
                </tr>
              </tbody>
            </table>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`Picker format by locale date 1`] = `
<div
  class="ant-picker"
>
  <div
    class="ant-picker-input"
  >
    <input
      placeholder="请选择日期"
      readonly=""
      size="12"
      title="2000-01-01"
      value="2000-01-01"
    />
    <span
      class="ant-picker-suffix"
    >
      <span
        aria-label="calendar"
        class="anticon anticon-calendar"
        role="img"
      >
        <svg
          aria-hidden="true"
          class=""
          data-icon="calendar"
          fill="currentColor"
          focusable="false"
          height="1em"
          viewBox="64 64 896 896"
          width="1em"
        >
          <path
            d="M880 184H712v-64c0-4.4-3.6-8-8-8h-56c-4.4 0-8 3.6-8 8v64H384v-64c0-4.4-3.6-8-8-8h-56c-4.4 0-8 3.6-8 8v64H144c-17.7 0-32 14.3-32 32v664c0 17.7 14.3 32 32 32h736c17.7 0 32-14.3 32-32V216c0-17.7-14.3-32-32-32zm-40 656H184V460h656v380zM184 392V256h128v48c0 4.4 3.6 8 8 8h56c4.4 0 8-3.6 8-8v-48h256v48c0 4.4 3.6 8 8 8h56c4.4 0 8-3.6 8-8v-48h128v136H184z"
          />
        </svg>
      </span>
    </span>
    <span
      class="ant-picker-clear"
    >
      <span
        aria-label="close-circle"
        class="anticon anticon-close-circle"
        role="img"
      >
        <svg
          aria-hidden="true"
          class=""
          data-icon="close-circle"
          fill="currentColor"
          focusable="false"
          height="1em"
          viewBox="64 64 896 896"
          width="1em"
        >
          <path
            d="M512 64C264.6 64 64 264.6 64 512s200.6 448 448 448 448-200.6 448-448S759.4 64 512 64zm165.4 618.2l-66-.3L512 563.4l-99.3 118.4-66.1.3c-4.4 0-8-3.5-8-8 0-1.9.7-3.7 1.9-5.2l130.1-155L340.5 359a8.32 8.32 0 01-1.9-5.2c0-4.4 3.6-8 8-8l66.1.3L512 464.6l99.3-118.4 66-.3c4.4 0 8 3.5 8 8 0 1.9-.7 3.7-1.9 5.2L553.5 514l130 155c1.2 1.5 1.9 3.3 1.9 5.2 0 4.4-3.6 8-8 8z"
          />
        </svg>
      </span>
    </span>
  </div>
</div>
`;

exports[`Picker format by locale dateTime 1`] = `
<div
  class="ant-picker"
>
  <div
    class="ant-picker-input"
  >
    <input
      placeholder="请选择日期"
      readonly=""
      size="21"
      title="2000-01-01 00:00:00"
      value="2000-01-01 00:00:00"
    />
    <span
      class="ant-picker-suffix"
    >
      <span
        aria-label="calendar"
        class="anticon anticon-calendar"
        role="img"
      >
        <svg
          aria-hidden="true"
          class=""
          data-icon="calendar"
          fill="currentColor"
          focusable="false"
          height="1em"
          viewBox="64 64 896 896"
          width="1em"
        >
          <path
            d="M880 184H712v-64c0-4.4-3.6-8-8-8h-56c-4.4 0-8 3.6-8 8v64H384v-64c0-4.4-3.6-8-8-8h-56c-4.4 0-8 3.6-8 8v64H144c-17.7 0-32 14.3-32 32v664c0 17.7 14.3 32 32 32h736c17.7 0 32-14.3 32-32V216c0-17.7-14.3-32-32-32zm-40 656H184V460h656v380zM184 392V256h128v48c0 4.4 3.6 8 8 8h56c4.4 0 8-3.6 8-8v-48h256v48c0 4.4 3.6 8 8 8h56c4.4 0 8-3.6 8-8v-48h128v136H184z"
          />
        </svg>
      </span>
    </span>
    <span
      class="ant-picker-clear"
    >
      <span
        aria-label="close-circle"
        class="anticon anticon-close-circle"
        role="img"
      >
        <svg
          aria-hidden="true"
          class=""
          data-icon="close-circle"
          fill="currentColor"
          focusable="false"
          height="1em"
          viewBox="64 64 896 896"
          width="1em"
        >
          <path
            d="M512 64C264.6 64 64 264.6 64 512s200.6 448 448 448 448-200.6 448-448S759.4 64 512 64zm165.4 618.2l-66-.3L512 563.4l-99.3 118.4-66.1.3c-4.4 0-8-3.5-8-8 0-1.9.7-3.7 1.9-5.2l130.1-155L340.5 359a8.32 8.32 0 01-1.9-5.2c0-4.4 3.6-8 8-8l66.1.3L512 464.6l99.3-118.4 66-.3c4.4 0 8 3.5 8 8 0 1.9-.7 3.7-1.9 5.2L553.5 514l130 155c1.2 1.5 1.9 3.3 1.9 5.2 0 4.4-3.6 8-8 8z"
          />
        </svg>
      </span>
    </span>
  </div>
</div>
`;

exports[`Picker format by locale month 1`] = `
<div
  class="ant-picker"
>
  <div
    class="ant-picker-input"
  >
    <input
      placeholder="请选择月份"
      readonly=""
      size="12"
      title="2000-01"
      value="2000-01"
    />
    <span
      class="ant-picker-suffix"
    >
      <span
        aria-label="calendar"
        class="anticon anticon-calendar"
        role="img"
      >
        <svg
          aria-hidden="true"
          class=""
          data-icon="calendar"
          fill="currentColor"
          focusable="false"
          height="1em"
          viewBox="64 64 896 896"
          width="1em"
        >
          <path
            d="M880 184H712v-64c0-4.4-3.6-8-8-8h-56c-4.4 0-8 3.6-8 8v64H384v-64c0-4.4-3.6-8-8-8h-56c-4.4 0-8 3.6-8 8v64H144c-17.7 0-32 14.3-32 32v664c0 17.7 14.3 32 32 32h736c17.7 0 32-14.3 32-32V216c0-17.7-14.3-32-32-32zm-40 656H184V460h656v380zM184 392V256h128v48c0 4.4 3.6 8 8 8h56c4.4 0 8-3.6 8-8v-48h256v48c0 4.4 3.6 8 8 8h56c4.4 0 8-3.6 8-8v-48h128v136H184z"
          />
        </svg>
      </span>
    </span>
    <span
      class="ant-picker-clear"
    >
      <span
        aria-label="close-circle"
        class="anticon anticon-close-circle"
        role="img"
      >
        <svg
          aria-hidden="true"
          class=""
          data-icon="close-circle"
          fill="currentColor"
          focusable="false"
          height="1em"
          viewBox="64 64 896 896"
          width="1em"
        >
          <path
            d="M512 64C264.6 64 64 264.6 64 512s200.6 448 448 448 448-200.6 448-448S759.4 64 512 64zm165.4 618.2l-66-.3L512 563.4l-99.3 118.4-66.1.3c-4.4 0-8-3.5-8-8 0-1.9.7-3.7 1.9-5.2l130.1-155L340.5 359a8.32 8.32 0 01-1.9-5.2c0-4.4 3.6-8 8-8l66.1.3L512 464.6l99.3-118.4 66-.3c4.4 0 8 3.5 8 8 0 1.9-.7 3.7-1.9 5.2L553.5 514l130 155c1.2 1.5 1.9 3.3 1.9 5.2 0 4.4-3.6 8-8 8z"
          />
        </svg>
      </span>
    </span>
  </div>
</div>
`;

exports[`Picker format by locale week 1`] = `
<div
  class="ant-picker"
>
  <div
    class="ant-picker-input"
  >
    <input
      placeholder="请选择周"
      readonly=""
      size="12"
      title="1999-52周"
      value="1999-52周"
    />
    <span
      class="ant-picker-suffix"
    >
      <span
        aria-label="calendar"
        class="anticon anticon-calendar"
        role="img"
      >
        <svg
          aria-hidden="true"
          class=""
          data-icon="calendar"
          fill="currentColor"
          focusable="false"
          height="1em"
          viewBox="64 64 896 896"
          width="1em"
        >
          <path
            d="M880 184H712v-64c0-4.4-3.6-8-8-8h-56c-4.4 0-8 3.6-8 8v64H384v-64c0-4.4-3.6-8-8-8h-56c-4.4 0-8 3.6-8 8v64H144c-17.7 0-32 14.3-32 32v664c0 17.7 14.3 32 32 32h736c17.7 0 32-14.3 32-32V216c0-17.7-14.3-32-32-32zm-40 656H184V460h656v380zM184 392V256h128v48c0 4.4 3.6 8 8 8h56c4.4 0 8-3.6 8-8v-48h256v48c0 4.4 3.6 8 8 8h56c4.4 0 8-3.6 8-8v-48h128v136H184z"
          />
        </svg>
      </span>
    </span>
    <span
      class="ant-picker-clear"
    >
      <span
        aria-label="close-circle"
        class="anticon anticon-close-circle"
        role="img"
      >
        <svg
          aria-hidden="true"
          class=""
          data-icon="close-circle"
          fill="currentColor"
          focusable="false"
          height="1em"
          viewBox="64 64 896 896"
          width="1em"
        >
          <path
            d="M512 64C264.6 64 64 264.6 64 512s200.6 448 448 448 448-200.6 448-448S759.4 64 512 64zm165.4 618.2l-66-.3L512 563.4l-99.3 118.4-66.1.3c-4.4 0-8-3.5-8-8 0-1.9.7-3.7 1.9-5.2l130.1-155L340.5 359a8.32 8.32 0 01-1.9-5.2c0-4.4 3.6-8 8-8l66.1.3L512 464.6l99.3-118.4 66-.3c4.4 0 8 3.5 8 8 0 1.9-.7 3.7-1.9 5.2L553.5 514l130 155c1.2 1.5 1.9 3.3 1.9 5.2 0 4.4-3.6 8-8 8z"
          />
        </svg>
      </span>
    </span>
  </div>
</div>
`;
